### PR TITLE
Update pagination on instances apis

### PR DESCRIPTION
**Pull request by juliasaveliff**
- Update /get-many 
- Implement /pages

### CODE REVIEW

File: rackcity/views/it_instance_views.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from rest_framework.parsers import JSONParser
 from rest_framework.pagination import PageNumberPagination
 from http import HTTPStatus
+import math
```
Comment (juliasaveliff) [inline rackcity/views/it_instance_views.py]: I ❤️math

File: rackcity/views/it_instance_views.py [hunk 2]
```diff
@@ -80,5 +106,25 @@ def instance_add(request):
...
+        status=HTTPStatus.BAD_REQUEST,
     )
+
+
+@api_view(['GET'])
+@permission_classes([IsAuthenticated])
+def instance_page_count(request):
+    """
+    Return total number of pages according to page size, which must be
+    specified as query parameter.
```
Comment (juliasaveliff) [inline rackcity/views/it_instance_views.py]: to be consistent with instance_page view

File: rackcity/views/it_instance_views.py (hunk 1 above)
Comment (benauriemma) [reply rackcity/views/it_instance_views.py]: Julia Math

File: rackcity/views/it_instance_views.py [hunk 3]
```diff
@@ -80,5 +106,25 @@ def instance_add(request):
...
+    if not request.query_params.get('page_size') or int(request.query_params.get('page_size')) <= 0:
+        return JsonResponse(
+            {"failure_message": "Must specify positive integer page_size."},
+            status=HTTPStatus.BAD_REQUEST,
+        )
+
+    page_size = int(request.query_params.get('page_size'))
+    instance_count = ITInstance.objects.all().count()
+    page_count = math.ceil(instance_count / page_size)
+    return JsonResponse({"pages": page_count})
```
Comment (benauriemma) [inline rackcity/views/it_instance_views.py]: This is picky but would it be more clear to say `{"page_count": page_count}` or `{"num_pages": page_count}`? I'll approve anyway

Comment (juliasaveliff) [reply rackcity/views/it_instance_views.py]: Very much agree changing

**Pull request merged**